nsds_lab_to_nwb/common/io.py

```diff
@@ -42,3 +44,16 @@ def csv_to_dict(csv_file):
     if ('key', 'value') in mydict.items():
         mydict.pop('key')
     return mydict
+
+
+# --- other file types ---
+
+def read_mat_file(file_path):
+    try:
+        f = h5py.File(file_path, 'r')
+    except OSError:
+        # if we get 'OSError: Unable to open file (File signature not found)'
+        # this mat file may be from an earlier MATLAB version
+        # and is not in HDF5 format.
+        f = scipy.io.loadmat(file_path)
+    return f
```

Comment:
I'd be in favor of converting the various .mat files into a standard format, removing this, and just having per-stimulus type readers in `stim_value_extractor.py`. If things are working now, feel free to ignore for this PR and I'll make an issue.